pyro/infer/autoguide/guides.py

```diff
@@ -463,23 +467,24 @@ class AutoMultivariateNormal(AutoContinuous):
         guide = AutoMultivariateNormal(model)
         svi = SVI(model, guide, ...)
 
-    By default the mean vector is initialized to zero and the Cholesky factor
-    is initialized to the identity.  To change this default behavior the user
-    should call :func:`pyro.param` before beginning inference, e.g.::
-
-        latent_dim = 10
-        pyro.param("auto_loc", torch.randn(latent_dim))
-        pyro.param("auto_scale_tril", torch.tril(torch.rand(latent_dim)),
-                   constraint=constraints.lower_cholesky)
+    By default the mean vector is initialized by ``init_loc_fn()`` and the
+    Cholesky factor is initialized to the identity.
     """
 
+    def __init__(self, model, prefix="auto", init_loc_fn=init_to_median,
+                 init_scale=1.0):
+        if not isinstance(init_scale, float) or not (init_scale > 0):
```

Comment:
This is going to be super convenient. Can we also add `init_scale` to the docstring?

Reply:
done.